src/style/rStats.css

```diff
@@ -1,80 +1,76 @@
 .rs-base {
-  position: absolute;
-  z-index: 10000;
-  padding: 10px;
-  background-color: #ed134a;
-  width: 300px;
-  font: 300 10px monospace;
+  background-color: #EF2D5E;
+  border-radius: 0;
+  font: 10px monospace;
   left: 5px;
-  top: 5px;
+  line-height: 1.2em;
```

Comment:
nit: `line-height: 1;` would fix it so the the bars vertically align correctly with the text